.github/workflows/ci.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       uses: arduino/setup-protoc@master
```

Comment:
should we lock to this version of the action so we dont leak the token to whatever lands in their master tomorrow?

i think `arduino/setup-protoc@v1.1.0` will work? if not we'll need to ask them to rls

Reply:
jk we can do a specific commit: `arduino/setup-protoc@7ad700d`

thx https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsuses

Reply:
oh that's so cool! it never occurred to me that github actions can keep secrets from being in the environment already unlike travis. that's dope :)